src/diffusers/loaders/unet.py

```diff
@@ -561,10 +561,104 @@ def _unfuse_lora_apply(self, module):
             if isinstance(module, BaseTunerLayer):
                 module.unmerge()
 
+    def _expand_lora_scales_dict(
+        self, scales, blocks_with_transformer: Dict[str, int], transformer_per_block: Dict[str, int]
+    ):
+        """
+        Expands the inputs into a more granular dictionary. See the example below for more details. 
+
+        Parameters:
+            blocks_with_transformer (`Dict[str, int]`):
+                Dict with keys 'up' and 'down', showing which blocks have transformer layers
+            transformer_per_block (`Dict[str, int]`):
+                Dict with keys 'up' and 'down', showing how many transformer layers each block has
+
+        E.g. turns
+            {
+                'down': 2,
+                'mid': 3,
+                'up': {
+                    'block_1': 4,
+                    'block_2': [5, 6, 7]
```

Comment:
did I understand it wrong or it is a typo?
```suggestion
                    'block_0': 4,
                    'block_1': [5, 6, 7]
```

Reply:
It's a typo. Have fixed it.